gno.land/pkg/gnoclient/mock_test.go

```diff
@@ -30,11 +31,11 @@ func (m *mockSigner) Sign(cfg SignCfg) (*std.Tx, error) {
 	return nil, nil
 }
 
-func (m *mockSigner) Info() keys.Info {
+func (m *mockSigner) Info() (keys.Info, error) {
 	if m.info != nil {
 		return m.info()
 	}
-	return nil
+	return nil, errors.New("empty key info")
```

Comment:
The default return for mocks here should be `return nil, nil`
```suggestion
	return nil, nil
```